components/CollectionItem.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Heading, HStack, Link, Stack, Text } from '@chakra-ui/react'
+import { ServicesList } from 'models'
+import React from 'react'
+import { MoreVertical } from 'react-feather'
+import { formatDate } from 'utils'
+import { TaxonomySection } from './TaxonomySection'
+
+interface CollectionItemProps {
+  servicesList: ServicesList
+}
+
+const CollectionItem = ({
+  servicesList,
+  ...props
+}: CollectionItemProps): JSX.Element => {
+  const taxonomies = [...new Set(servicesList?.taxonomies)]
+
+  return (
+    <Box
+      {...props}
+      boxShadow="md"
+      rounded="lg"
+      p={{ base: 4, md: 8 }}
+      _hover={{ boxShadow: 'lg' }}
+    >
```

Comment:
We should style `CollectionItem` [the same as `ResourceCard`](https://github.com/MutualAidNYC/services-lists/blob/main/components/ResourceCard.tsx#L22) for consistency - the `boxShadow`, `borderRadius`, etc should be in Figma.

Reply:
I switched the base shadow and other props to the one for `ResourceCard` but I kept the hover shadow state because we don't have a hover state for when you hover over a card in the designs. Think we can keep this for now and maybe create a common card component with a preset hover style when we have designs.